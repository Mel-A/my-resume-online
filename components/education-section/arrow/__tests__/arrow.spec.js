import { shallow } from 'enzyme'
import Arrow from '..'

test('arrow renders correctly', () => {
  const arrow = shallow(<Arrow />)
  expect(arrow).toMatchSnapshot()
});