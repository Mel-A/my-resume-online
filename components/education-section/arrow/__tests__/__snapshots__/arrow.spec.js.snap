// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`arrow arrow renders correctly 1`] = `ShallowWrapper {}`;

exports[`arrow arrow renders theme correctly 1`] = `ReactWrapper {}`;
